src/components/video-capture/index.js

```diff
@@ -0,0 +1,80 @@
+module.exports = {
+    template: require('video-capture.html'),
+    data: function() {
+        return {
+            buttonLabel: "start capture",
+            captureFilename: null,
+            timesliceInMs: 5000,
+            fps: 3,
```

Comment:
this sounds like a very low framerate?

Reply:
It seems to be ignored anyway. I think this PR should be left in stasis until MacOS implementation is available.